tests/scale_test/utils.py

```diff
@@ -0,0 +1,238 @@
+import json
+import uuid
+import requests
+import click
+
+
+def send_notification(lira_url, auth_token, notification):
+    """Send a notification to a given Lira.
+
+    Args:
+        lira_url (str): A typical Lira url, e.g. https://pipelines.dev.data.humancellatlas.org/
+        auth_token (str): The Lira auth_token.
+        notification (dict): A dict of notification content.
+
+    Returns:
+        requests.Response: The response object returned by Lira.
+    """
+    response = requests.post(url=lira_url + 'notifications', json=notification, params={'auth': auth_token})
+    return response
+
+
+def prepare_notification(bundle_uuid, bundle_version, subscription_id, label=None, transaction_id=None):
+    """Compose the notification content from given values.
+
+    Args:
+        bundle_uuid (str): A valid Blue Box bundle uuid.
+        bundle_version (str): A valid Blue Box bundle version.
+        subscription_id (str): A valid Lira subscription id in Blue Box.
+        label (str): A label to be added to the notification, which will then be added to the workflow started by Lira.
+        transaction_id (str): A valid transcription id.
```

Comment:
I think you meant `transaction` here instead of `transcription`!

Reply:
Oops, dumb typo, I must be reading the bioinformatics books at that time lol. Nice catch!